open-sphere-base/core/src/main/java/io/opensphere/core/help/AboutDialog.java

```diff
@@ -47,13 +49,19 @@
     /** The software version. */
     private String myVersion;
 
+    /** The toolbox through which application state is accessed. */
+    private final Toolbox myToolbox;
+
     /**
      * Default Constructor.
+     *
+     * @param toolbox The toolbox through which application state is accessed.
      */
-    public AboutDialog()
+    public AboutDialog(Toolbox toolbox)

```

Comment:
Could this be removed now that the toolbox doesn't need to be passed through all over?

Reply:
fixed